src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -149,6 +153,21 @@ public LoggerConfiguration ToMaximumDepth(int maximumDestructuringDepth)
             _setMaximumDepth(maximumDestructuringDepth);
             return _loggerConfiguration;
         }
+
+        /// <summary>
+        /// When destructuring objects, string values can be restricted to specified length
+        /// thus avoiding bloating payload. Limit is applied to each value separately, 
+        /// sum of length of strings can exceed limit.
+        /// </summary>
+        /// <param name="maximumStringLength">The maximum string length.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">When passed length is less or equal to 2</exception>
+        public LoggerConfiguration ToMaximumStringLength(int maximumStringLength)
+        {
+            if (maximumStringLength < 2) throw new ArgumentOutOfRangeException(nameof(maximumStringLength), maximumStringLength, "Maximum string length have to be greater than 1");
```

Comment:
Exception messages in Serilog are all sentences, so needs a `.` at the end of the message. Perhaps a slight rewording `"Maximum string length must be at least two."` ?